ts/WoltLabSuite/Core/Acp/Ui/User/Action/Handler/Ban.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @author  Joshua Ruesweg
+ * @copyright  2001-2021 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Acp/Ui/User/Action/Handler
+ * @since       5.5
+ */
+
+import * as Ajax from "../../../../../Ajax";
+import BanDialog from "./Dialog/Ban";
```

Comment:
I think I prefer `./Ban/Dialog` instead.

Reply:
I would, actually. However, I shy away from creating a subfolder for every possible dialogue. Especially if the actual action is in a folder above and not in the corresponding folder.